Panto/index.html

```diff
@@ -68,7 +75,7 @@ <h3>Luxury facilities</h3>
               </p>
               <div class="more-info-box">
                 <a href="#" class="more-info">More Info</a>
-                <span class="more-info-arrow">&rarr;</span>
+                <a href="#" class="more-info-arrow">&rarr;</a>
```

Comment:
redundant `a` tag
wrap them both inside a single `a` tag
having 2 different anchor tags for a single link would cause out of sync updates in the future
imagine changing the link for the text (More Info) but forgetting to update the link for the arrow